src/openrct2/windows/tile_inspector.h

```diff
@@ -12,20 +12,24 @@
 #include "../common.h"
 #include "../interface/Window.h"
 
-enum TILE_INSPECTOR_PAGE
+enum class TileInspectorPage : int16_t
 {
-    TILE_INSPECTOR_PAGE_DEFAULT,
-    TILE_INSPECTOR_PAGE_SURFACE,
-    TILE_INSPECTOR_PAGE_PATH,
-    TILE_INSPECTOR_PAGE_TRACK,
-    TILE_INSPECTOR_PAGE_SCENERY,
-    TILE_INSPECTOR_PAGE_ENTRANCE,
-    TILE_INSPECTOR_PAGE_WALL,
-    TILE_INSPECTOR_PAGE_LARGE_SCENERY,
-    TILE_INSPECTOR_PAGE_BANNER,
-    TILE_INSPECTOR_PAGE_CORRUPT
+    Default = 0,
+    Surface,
+    Path,
+    Track,
+    Scenery,
+    Entrance,
+    Wall,
+    LargeScenery,
+    Banner,
+    Corrupt
 };
 
+std::underlying_type_t<TileInspectorPage> toIndex(TileInspectorPage page) {
+    return static_cast<std::underlying_type_t<TileInspectorPage>>(page);
+}
+
```

Comment:
We already have this in `src\openrct2\util\Util.h`:
https://github.com/OpenRCT2/OpenRCT2/blob/84c9ff833a778744c620cf7c4a56fa4eacfa2ac9/src/openrct2/util/Util.h#L84-L87

Reply:
Will use that one :+1: